apps/studio/components/interfaces/Auth/index.tsx

```diff
@@ -2,5 +2,8 @@ export { default as AdvancedAuthSettingsForm } from './AdvancedAuthSettingsForm/
 export { default as AuthProvidersForm } from './AuthProvidersForm/AuthProvidersForm'
 export { default as BasicAuthSettingsForm } from './BasicAuthSettingsForm/BasicAuthSettingsForm'
 export { default as EmailTemplates } from './EmailTemplates/EmailTemplates'
+export { default as MfaAuthSettingsForm } from './MfaAuthSettingsForm/MfaAuthSettingsForm'
```

Comment:
this one's more of a code style nit

we're following a pattern of directly exporting the component from the file itself - so no longer need `index.ts` files to expose the components like this

e.g `export const MfaAuthSettingsForm = () => ...` from `MfaAuthSettingsForm.tsx`
And no longer need `export default ...` too